local variable referenced before assignment

```
args = (), arguments = '', tb_line = ''

    def wrapper(*args):
        arguments = self._convert_args(args)
        if docstring:
            self.logger.debug(docstring)
        self.channel.send("%s(%s)" % (name, arguments))
        try:
            return self.channel.receive()
        except Exception as error:
            # Error will come as a string of a traceback, remove everything
            # up to the actual exception since we do get garbage otherwise
            # that points to non-existent lines in the compiled code
            for tb_line in reversed(str(error).split('\n')):
                if tb_line:
                    exc_line = tb_line
                    break
>           raise RuntimeError(exc_line)
E           UnboundLocalError: local variable 'exc_line' referenced before assignment

.venv/lib/python3.6/site-packages/remoto/connection.py:126: UnboundLocalError
```

It seems that `exc_line` is not initiated

--- a/remoto/connection.py
+++ b/remoto/connection.py
@@ -119,6 +119,7 @@ class ModuleExecute(object):
                 # Error will come as a string of a traceback, remove everything
                 # up to the actual exception since we do get garbage otherwise
                 # that points to non-existent lines in the compiled code
+                exc_line = str(error)
                 for tb_line in reversed(str(error).split('\n')):
                     if tb_line:
                         exc_line = tb_line

